kratos/utilities/rve_periodicity_utility.cpp

```diff
@@ -15,7 +15,8 @@
 // External includes
 
 // Project includes
-#include "rve_periodicity_utility.h"
+#include "includes/model_part.h"
```

Comment:
This is not correct. It only works because you always include the model_part.h before the rve_periodicity_utility.h but the header needs the definition of the ModelPart because it uses it in the creator and some function declarations.